src/components/QuestionPrompt.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const QuestionPrompt = (props) => {
+  return (
+    <div className="squestion-prompt" role="presentation">
```

Comment:
Isn't this supposed to be `squestion-prompt`? The new stylesheet specifies `.question-prompt`, and this classname seems to break the style of the Question Prompt.

Reply:
Oooo. Duh. Yeah, that's a typo I should've caught.